discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -152,8 +157,16 @@ def download_infra_env_image(self, iso_download_path = None) -> None:
         self._infra_env.download_image(iso_download_path=iso_download_path)
 
     @JunitTestCase()
-    def generate_and_download_infra_env(self, iso_download_path = None, static_network_config=None, iso_image_type=None, ssh_key=None, ignition_info=None):
-        self.generate_infra_env(static_network_config=static_network_config, iso_image_type=iso_image_type, ssh_key=ssh_key, ignition_info=ignition_info)
+    def generate_and_download_infra_env(
```

Comment:
I think we should add type annotations here

Reply:
The already defined function had no annotations, so I'm eager to say that we should just make this PR as small as possible in order to get the stuff working again and do the refactoring in the follow-up PR. With the CI broken already for a week I'd rather fix it as soon as possible with as small changes as possible

Reply:
ack. will add annotations in seperate PR